### PR TITLE
Addition of directives/controllers in index.html

### PR DESCRIPTION
**Pull request by renujhamtani**
@ardasgup @pbathia  Please review. I will merge once this reviewed. This PR was to add some missing controllers and directives in index.html. Although we need to always test it through chromed.html, sometimes it is easy to first develop on unchromed version and then test on chromed version. Hence pushing this file.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -98,23 +102,26 @@
...
     <script src="cases/controllers/exportCSVButton.js"></script>
+    <script src="cases/controllers/filterSelect.js"></script>
     <script src="cases/controllers/group.js"></script>
     <script src="cases/controllers/groupList.js"></script>
     <script src="cases/controllers/groupSelect.js"></script>
     <script src="cases/controllers/list.js"></script>
     <script src="cases/controllers/listAttachments.js"></script>
     <script src="cases/controllers/listFilter.js"></script>
+    <script src="cases/controllers/topCases.js"></script>
+    <script src="cases/controllers/editAscension.js"></script>
```
Comment (pbathia) [inline app/index.html]: @renujhamtani I dont have this file..is this you are going to add ?

**Comment by renujhamtani** [comment]
@ardasgup @pbathia removed the unused/not present directive/controller. Please review it.

**Comment by pbathia** [comment]
@renujhamtani  Looks good to me.

**Comment by ardasgup** [comment]
Reviewed!

**Pull request merged**